### ISSUE TITLE
input image size

### ISSUE THREAD

**Issue by bhack** [UX-Decoder/Semantic-SAM#7]
Are we enforced to resize the input image `1024x1024` at inference?

Is there a way to inference at higher resolutions without loosing details in preprocessing?

**Comment by FengLi-ust** [comment]
No, we are not enforced to resize to `1024x1024`. You can use other image sizes. Slightly increasing the image size may help improve the performance, however, as the training if for `1024x1024` image size, using a very large image size may hurt the performance. You can finetune the model with a large image size.

**Comment by bhack** [comment]
> You can finetune the model with a large image size.

Any implicit constraint for the aspect ratio?

**Comment by Yaziwel** [comment]
I have a similar issue. My input images have an original size of 128x128, and I obtain the best segmentation results when I resize the input images to 1024x1024. However, the output masks are also of size 1024, and I want to align the mask size with the input size. How can I adjust the mask size to align it with the input size? Just simply resize them?
